src/taipy/core/_orchestrator/_orchestrator.py

```diff
@@ -295,7 +297,6 @@ def _fail_subsequent_jobs(cls, failed_job: Job):
                 cls.__find_subsequent_jobs(failed_job.submit_id, set(failed_job.task.output.keys()))
             )
             for job in to_fail_or_abandon_jobs:
-                print(f"Abandoning job: {job.id}")
```

Comment:
Can we keep this message?
Or log it instead of `print`?

Or when the job is abandoned, we log a message saying why the job is abandoned. It's better for the user to understand the problem, but probably quite tricky.

Reply:
this was a print that I used when debugging an issue that I forgot to remove :D

Reply:
We can log a message when canceling or abandoning a job in the method `_orchestrator._cancel_job`.

Reply:
We have already log that information https://github.com/Avaiga/taipy-core/blob/develop/src/taipy/core/_orchestrator/_orchestrator.py#L245

Reply:
No log in the else block which correspond to the case where we do cancel and abandon jobs.